src/data/users.js

```diff
@@ -188,6 +188,12 @@ const users = [
     infoLink: "https://epilot.cloud/",
     pinned: true,
   },
+  {
+    caption: "Tricentis",
+    image: "img/tricentis-logo.svg",
+    infoLink: "https://www.tricentis.com/",
+    pinned: true,
```

Comment:
No more pinned without donations

```suggestion
    pinned: false,
```